blockchain/fundamentals/the-blockchain/what-is-a-blockchain.md

```diff
@@ -37,19 +37,19 @@ links:
 ---
 ## Content
 
-A **blockchain** (a literal chain of blocks) is a peer-to-peer network which is looking to build and maintain a sequence of information bundles. For the information to be added to the sequence, it has to be verified first by decentralized nodes.
+A **blockchain** (a literal chain of blocks) is a peer-to-peer network aiming to build and maintain a sequence of information bundles. For the information to be added to the sequence, it has to be verified first by decentralized nodes.
```

Comment:
Although it's an improvement to the original "which is looking" I'd probably say "designed" is even more accurate than "aiming"

> A **blockchain** (a literal chain of blocks) is a peer-to-peer network designed to build and maintain a sequence of information bundles.

Reply:
cool